modules/ROOT/pages/to-find-info.adoc

```diff
@@ -30,54 +13,141 @@ Exchange Search looks for terms in:
+
+New Exchange search experience:
+
+image::search.png[New Exchange search experience]
+
+== Tags, Categories, and Ranking in the New Exchange Search Experience
+
+To filter search results by tags and categories in the new Exchange search experience, use the drop-down menus for tags and categories on the search page.
 
-You can also search by clicking Category or Tag elements. This creates a filtered search you can use to find similar assets.
+To select the ranking of search results in the new Exchange search experience, use the *Sort by* drop-down menu to choose *Relevance* (default), *Name*, *Last modified*, or *Rating*. *Relevance* sorts the matching results by where a match is found in the asset metadata, such as name, title, pages, and API specification metadata. API specification metadata includes endpoints and types.
 
-In Exchange Private Cloud Edition and GovCloud, searches apply only to the Anypoint Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
+== Searching in Organizations
 
-In Exchange Cloud, you can select which of the organizations visible to you should be included in each search.
+In the original Exchange search experience, searches apply only to the Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
+
+In the new Exchange search experience, you can select which of the organizations visible to you should be included in each search. By default, no organization is selected in the drop-down menu, and Exchange searches MuleSoft's public assets and the assets of the master organization and its business groups.
 
 == Save a Search
 
-. Enter a search term and click Save This Search.
-. Exchange prompts you to give the search a name, and to select if the search is for your organization or personal. Administrators can save searches visible to anyone in their current business group. Personal searches are visible only to the person who creates them.
-. View the left navigation bar to see the saved searches. Exchange lists search terms specific to your business group. Below those terms are saved searches you created.
+The original Exchange search experience and the new Exchange search experience save searches in the same way.
+
+. Enter a search term and click *Save This Search*.
+. Exchange prompts you to give the search a name, and to select if the search is for your organization or personal.
++
+Administrators can save searches visible to anyone in their current business group. Personal searches are visible only to the person who creates them.
+. View the left navigation bar to see the saved searches.
```

Comment:
Is there another save step after providing the name and type of search?

Reply:
Should this step be a confirmation that your search was added, rather than just viewing?

Reply:
Updated.